FreeRTOS/FreeRTOS
# Add MQTT_Mutual_Auth_wolfSSL demo

### Add MQTT_Mutual_Auth_wolfSSL demo ###

Description
------------
This demo shows an MQTT subscriber / publisher establishing a TLS connection with an MQTT broker to send and receive topics in encrypted form. wolfSSL acts as a TLS library under the FreeRTOS MQTT library. 

Test Steps
-----------

1. Double-click the solution file named "mqtt_mutual_auth_demo_wolfSSL.sln" included in this folder to start Visual Studio.

1. Follow the instructions in README_wolfSSL.md to set up the demo. 

1. Build the RTOSDemo project.

1. Run the RTOSDEmo.exe with debugger.




Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
